### PR TITLE
Upgrade ONNX to 1.13

### PR DESCRIPTION
**Pull request by p-wysocki**
Signed-off-by: p-wysocki <przemyslaw.wysocki@intel.com>

ONNX 1.13 has been released: https://github.com/onnx/onnx/releases/tag/v1.13.0

Reference PR: https://github.com/conda-forge/onnx-feedstock/pull/1

### CODE REVIEW

**Comment by xhochy** [comment]
`six` is no longer a runtime dependency. Can you remove that, please?

**Review by xhochy — APPROVED** [review]
Thanks!

**Comment by xhochy** [comment]
It looks like "410b39950367857f97b65093681fe2495a2e23d63777a8aceaf96c56a16d166e" is not the correct SHA256 for the referenced tarball.

**Comment by p-wysocki** [comment]
Shouldn't it be taken from here? https://pypi.org/project/onnx/#copy-hash-modal-ea1c53bf-d6a7-4628-8b08-d140605d3487

**Comment by xhochy** [comment]
No, this isn't building from the PyPI tarballs but the ones from Github (https://github.com/onnx/onnx/releases/tag/v1.13.0)

**Comment by p-wysocki** [comment]
I was certain it used PyPI. Thank you!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -42,7 +42,6 @@ requirements:
     - python
     - protobuf
```
Comment (xhochy) [inline recipe/meta.yaml:43]: Add the correct protobuf pins and remove the patch in line 10 (incl. the patch file)
```suggestion
    - protobuf >=3.20.2,<4
```

**Comment by p-wysocki** [comment]
Thank you for helping in the PR. It seems that the last three checks are failing due to an internal server error when trying to get `numpy==1.20.3` from [https://conda.anaconda.org/conda-forge/linux-aarch64/numpy-1.20.3-py38h6f99c5d_2.tar.bz2](https://conda.anaconda.org/conda-forge/linux-aarch64/numpy-1.20.3-py38h6f99c5d_2.tar.bz2%3E). Do you have an idea what could be the cause of this?

**Comment by xhochy** [comment]
> Thank you for helping in the PR. It seems that the last three checks are failing due to an internal server error when trying to get `numpy==1.20.3` from [https://conda.anaconda.org/conda-forge/linux-aarch64/numpy-1.20.3-py38h6f99c5d_2.tar.bz2](https://conda.anaconda.org/conda-forge/linux-aarch64/numpy-1.20.3-py38h6f99c5d_2.tar.bz2%3E). Do you have an idea what could be the cause of this?

There was a Cloudflare Downtime yesterday. I will restart the builds.

**Pull request merged**